src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,19 @@
+@import "./../../utils/variables.scss";
+
+.TodoInfo {
+  width: max-content;
+  padding: 8px;
+  margin: 20px 0;
+  border-radius: $-border-radius;
```

Comment:
Consider using more descriptive class names instead of `.TodoInfo`. For example, `.todo-info-container` or `.todo-info-wrapper`. 🤔